java/meterpreter/stdapi/src/main/java/com/metasploit/meterpreter/stdapi/stdapi_sys_process_getpid_V1_5.java

```diff
@@ -0,0 +1,49 @@
+package com.metasploit.meterpreter.stdapi;
+
+import com.metasploit.meterpreter.Meterpreter;
+import com.metasploit.meterpreter.TLVPacket;
+import com.metasploit.meterpreter.TLVType;
+import com.metasploit.meterpreter.command.Command;
+
+import java.lang.reflect.Method;
+
+public class stdapi_sys_process_getpid_V1_5 implements Command {
+    private static boolean classExists(String className) {
+        try {
+            Class.forName(className);
+            return true;
+        } catch (ClassNotFoundException e) {
+            return false;
+        }
+    }
+
+    public int execute(Meterpreter meterpreter, TLVPacket request, TLVPacket response) throws Exception {
+        if (classExists("java.lang.ProcessHandle"))
```

Comment:
It might be good to split this out into its own v1_9 implementation

https://github.com/rapid7/metasploit-payloads/blob/6e08d1f9812aa4d7a76b451fd5e0bae03975bb91/java/meterpreter/meterpreter/src/main/java/com/metasploit/meterpreter/CommandManager.java#L83-L106